### ISSUE TITLE
Server side changes

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/loopback-tutor-intern-8#45]
- [ ] update routes + controllers
- [ ] .env + moving stuff there...

**Comment by atherdon** [comment]
we can close this, right? it's a cool project improvement, right?

**Comment by chauhannishith** [comment]
yes it is better to have a task list compared to lots of issues, its easier to keep track of whats done
